site/content/blog/2023-10-12-maplocal-add-on/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Map Local Add-on"
+summary: "Allows mapping of responses to content of chosen local file."
+images:
+#- https://www.zaproxy.org/blog/2023-10-12-maplocal-add-on/images/image.png
+type: post
+tags:
+- blog
+- maplocal
+date: "2023-10-12"
+authors:
+    - Keindel (Andrey Maksimov)
+---
+
+This extension allows you to specify mappings of local resources to replace responses when using ZAP.
+Feature is an analog of:
+- Map Local in Charles https://www.charlesproxy.com/documentation/tools/map-local/
+- Map Local in Proxyman https://docs.proxyman.io/advanced-features/map-local
+
+## Installing the add-on
+
+You can install it via the [ZAP Marketplace](/addons/).
+
+## Using the Map Local
+
+1. RMB on Site or in History Tab to call context menu.
+2. Choose "MapLocal..."
+3. In the dialog window you can modify fields, if needed:  Protocol, Host, Path and Query
+4. Click on "Browse" button to choose local file for mapping or just provide the path to the field "Local Path"
+5. Save
+6. That's it! Now you can see newly created mapping in Map Local extension status panel 🎉
+   from here on all requests to the matching url's will return responses with the content of chosen local file
```

Comment:
Maybe include a screen shot.

A small video or animated gif of an example in action (before/after of a site)